.kokoro-autosynth/build-in-docker.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
This file is a copy of `build.sh` with edits to work correctly inside trampoline.

- bazel installation is omitted since that already exists in the Docker image gcr.io/gapic-images/googleapis
- nvm installation is omitted since it is only needed by NodeJS